test/rulesets/Base/base.jl

```diff
@@ -125,6 +125,24 @@
         )
     end
 
+    @testset "copy" for T in (Float64, ComplexF64)
+        test_frule(copy, randn(T))
+        test_frule(copy, randn(T, 4))
+        test_frule(
```

Comment:
do these work for you? I get
```julia
julia> copy((1.0, ))
ERROR: MethodError: no method matching copy(::Tuple{Float64})
Closest candidates are:
  copy(::LinearAlgebra.CholeskyPivoted) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.6/LinearAlgebra/src/cholesky.jl:440
  copy(::T) where T<:SHA.SHA3_CTX at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.6/SHA/src/types.jl:213
  copy(::LinearAlgebra.LQ) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.6/LinearAlgebra/src/lq.jl:113
  ...
Stacktrace:
 [1] top-level scope
   @ REPL[5]:1
```

Reply:
they don't actually. I'll add a test with dictionaries instead

Reply:
I didn't know how to add more sophisticated tests so I just removed the tuple test

Reply:
A simple test with dicts fails because `rand_tangent` is not defined on Dicts. I will open an issue